bitcash/network/rates.py

```diff
@@ -44,6 +44,14 @@
     ('sgd', 'Singapore Dollar'),
     ('thb', 'Thai Baht'),
     ('twd', 'New Taiwan Dollar'),
+    ('ḿxn', 'Mexican Peso'),
```

Comment:
Have you tested this currency? It's not working on my end.
Also it seems that special characters are not compliant with: https://en.wikipedia.org/wiki/ISO_4217

Reply:
There shouldn't be any special characters.

Reply:
> ('ḿxn'

You can see the m.

Reply:
Thank you!